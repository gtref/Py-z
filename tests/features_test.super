// Test file for advanced features

let a = 10;
let b = 20;

if (a < b) {
  print("a is less than b, which is correct.");
} else {
  print("a is not less than b, something is wrong.");
}

let counter = 0;
while (counter < 3) {
  print("Loop iteration:");
  print(counter);
  counter = counter + 1;
}

print("Test complete.");
