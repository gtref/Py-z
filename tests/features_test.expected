a is less than b, which is correct.
Loop iteration:
0
Loop iteration:
1
Loop iteration:
2
Test complete.
